app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -923,7 +914,6 @@ private void enterVRVideo(@VideoProjectionMenuWidget.VideoProjectionFlags int aP
         mProjectionMenu.setSelectedProjection(aProjection);
         // Backup the placement because the same widget is reused in FullScreen & MediaControl menus
         mProjectionMenuPlacement.copyFrom(mProjectionMenu.getPlacement());
-        this.setVisible(false);
```

Comment:
This should be added back, we need to make sure that the full-screen navigation bar is not visible when we play VR video.

https://github.com/Igalia/wolvic/assets/43995067/816d1f41-6686-4743-8d21-58bdf6b5dfc0

```suggestion
        this.setVisible(false);
```

Reply:
Interesting, I removed it because it was not making any difference to me, but from your video it's clear that it's still needed.